app/javascript/article-form/components/ClipboardButton.jsx

```diff
@@ -44,18 +44,13 @@ export const ClipboardButton = ({
       value={linksToMarkdownForm(imageUrls)}
     />
     <Button
-      className="spec__image-markdown-copy"
+      className="spec__image-markdown-copy whitespace-nowrap fw-normal"
       variant="ghost"
-      contentType="icon"
+      contentType="icon-left"
       icon={CopyIcon}
-    />
-    <span
-      id="image-markdown-copy-link-announcer"
-      role="alert"
-      className={`fs-s ${showCopyMessage ? '' : 'opacity-0'}`}
     >
-      Copied!
-    </span>
+      {showCopyMessage ? 'Copied!' : 'Copy...'}
```

Comment:
Why did this change as a part of this PR? (Also, it looks like this...[is already](https://github.com/forem/forem/blob/13ff4c7fbe3a070f6af9dda3c3dfa87e6f2badb7/app/javascript/article-form/components/ClipboardButton.jsx#L52) on master? 🤔)

Reply:
That's weird... https://github.com/forem/forem/pull/12128

Something went off while merging..?